parser/parser.y

```diff
@@ -47,6 +47,8 @@ import (
 %token	<ident>
 	/*yy:token "%c"     */	identifier      "identifier"
 	/*yy:token "\"%c\"" */	stringLit       "string literal"
+	singleAtIdentifier		"identifier with single leading at"
```

Comment:
It's some kind of workaround for the current lexer. the lexer will convert `user@host` into token `user` and `@host` while for `"user"@"host"` is `user`, `@`, `host`. differentiating `@host` and `@userVariable` in the lexer may require knowledge about the current parsing context, which is lacking now.